__tests__/endpoints.test.js

```diff
@@ -58,3 +58,38 @@ describe("/api", () => {
       });
   });
 });
+describe("/api/reviews/:review_id", () => {
+  test("GET - status 200, responds with review object with correct properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then((response) => {
+        const review = response.body.review;
+        review.forEach((review) => {
```

Comment:
Great work! 

For future development, maybe have a look at [.toMatchObject( )](https://jestjs.io/docs/expect#tomatchobjectobject) and [expect.objectContaining( )](https://jestjs.io/docs/expect#expectobjectcontainingobject)